### PR TITLE
Ardenian

### PR DESCRIPTION
**Pull request by Ardenian**
- Added new resource 'supplies': Having more supplies than humans will increase human count slowly up to houses*10. Having less supplies than humans will slowly decrease human count ( starving). Also added a button to gather supplies.
- Fixed a typo in index.html and changed directory paths

Known issue: 
```
this.increase = function ( amount ) {
    this.value += !Number.isNaN( amount ) ? amount : 1;
}
```
Does not work as intended right now. doing myResource.increase() will make it be NaN and not increase it by 1, as intended.

Some of the changes are quite rough and can be simplified, would be great if you could take a quick look on it @Idrinth

### CODE REVIEW

**Comment by Ardenian** [comment]
It is probably easier to take a look on the actual files and not the changes, formating quite overlights changes.

File: index.html [hunk 1]
```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="utf8"/>
     <title>No Survival - The Game</title>
-    <link href="/styles/layout.css" rel="stylesheet"/>
```
Comment (Idrinth) [inline index.html]: why is that change in?

Comment (Ardenian) [reply index.html]: I played around with styles and CSS and forgot to redo my playing around, sorry, adding it again

Comment (Ardenian) [reply index.html]: Oh, just checked, we changed it because my root is differently, think we talked about it in chat recently

Comment (Idrinth) [reply index.html]: the point was not that it wasn't required for your setup, still no reason to make it public imo.
This was a way to not setup a local server for you ;)

**Pull request merged**